src/makeCAF.C

```diff
@@ -446,7 +453,10 @@ void loop(CAF &caf,
   // figure out which triggers we need to loop over between the various reco fillers
   std::map<const cafmaker::IRecoBranchFiller*, std::deque<cafmaker::Trigger>> triggersByRBF;
   for (const std::unique_ptr<cafmaker::IRecoBranchFiller>& filler : recoFillers)
+  {
+    if (filler->GetName() == "LArTMSMatcher" || filler->GetName() == "LArMINERvAMatcher") continue; //We don't want to store a trigger from a Matcher algorithm
```

Comment:
Per the general comment, this would become

```suggestion
  {
    if (filler->FillerType != RecoFillerType::BaseReco) continue; //We don't want to store a trigger from a Matcher algorithm
```

Reply:
Implemented